theano/configdefaults.py

```diff
@@ -1237,39 +1251,38 @@ def default_blas_ldflags():
                     ['-l%s' % l for l in ["mk2_core", "mk2_intel_thread",
                                           "mk2_rt"]])
 
-        # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
-            # importable, then the libraries (installed by conda
-            # package "mkl-rt") are actually available.  Using
-            # "conda install mkl" will install both, as well as
-            # optimized versions of numpy and scipy.
-            try:
-                import mkl  # noqa
-            except ImportError as e:
-                _logger.info('Conda mkl is not available: %s', e)
+        # MKL
+        # If mkl can be imported then use it. On conda:
+        # "conda install mkl-service" installs the Python wrapper and
+        # the low-level C libraries as well as optimised version of
+        # numpy and scipy.
+        try:
+            import mkl  # noqa
+        except ImportError as e:
+           if any([m for m in ('conda', 'Continuum') if m in sys.version]):
+               _logger.warning('install mkl with `conda install mkl-service`: %s', e)
+        else:
+            # This branch is executed if no exception was raised
+            if sys.platform == "win32":
+                lib_path = os.path.join(sys.prefix, 'Library', 'bin')
```

Comment:
On windows, lib_path is a list and otherwise, it is a list. I thin it should always be a list, so wrap this in a list here.